lib/src/variant.dart

```diff
@@ -35,7 +35,7 @@ import 'win32/oleaut32.g.dart';
 
 sealed class _VARIANT_Anonymous_3 extends Struct {
   external Pointer pvRecord;
-  external Pointer<COMObject> pRecInfo;
+  external COMObject pRecInfo;
```

Comment:
Now that this isn't a pointer to `COMObject`, should the variable name be updated to `recInfo`?

Reply:
The `COMObject` struct has a single field called `lpVtbl`, representing a Pointer to the COM v-table: https://github.com/dart-windows/win32/blob/ce8927ce81eb972bde92aaf4a2742631e9aa75b0/lib/src/combase.dart#L18-L27 which makes this struct identical to `VTablePointer`.

Nevertheless, I should just change these to `VTablePointer` to make things clearer.